sys/include/net/gnrc/sixlowpan/frag/stats.h

```diff
@@ -35,6 +35,8 @@ typedef struct {
                              *   reassembly buffer is full */
     unsigned frag_full;     /**< counts the number of events that there where
                              *   no @ref gnrc_sixlowpan_frag_fb_t available */
```

Comment:
```suggestion
                             *   no @ref gnrc_sixlowpan_frag_fb_t available */
                             #if IS_USED(MODULE_GNRC_SIXLOWPAN_FRAG_STATS)
```

Reply:
This struct is only used with `gnrc_sixlowpan_frag_stats`. Otherwise, I would need to surround `rbuf_full` and `frag_full` as well.